src/main/host/memory_manager/memory_mapper.rs

```diff
@@ -64,7 +64,7 @@ fn log_regions<It: Iterator<Item = (Interval, Region)>>(level: log::Level, regio
 /// MemoryMapper via its `handle_*` methods.
 #[derive(Debug)]
 pub struct MemoryMapper {
-    memory_copier: MemoryCopier,
+    _memory_copier: MemoryCopier,
```

Comment:
I think we can just delete this field.

Reply:
Removed the field, and also the now unused `let memory_copier = MemoryCopier::new(thread.system_pid())`.